packages/ui/src/components/filters/CheckboxFilter.tsx

```diff
@@ -100,6 +108,39 @@ const CheckboxFilter = ({
         }
     }, [selectedFilters]);
 
+    const renderCheckBox = (filter: IFilter, i = -1) => (
+        <StackLayout
+            className={styles.checkboxFilterItem}
+            horizontal
+            key={`${filterGroup.field}-${filter.id}-${filter.data.count}-${localselectedFilters.length}-${i}`}
+        >
+            <Checkbox
+                checked={localselectedFilters.some((f) => f.data.key === filter.data.key)}
+                className={styles.fuiMcItemCheckbox}
+                id={`input-${filter.data.key}`}
+                name={`input-${filter.id}`}
+                onChange={(e) => {
+                    const { checked } = e.target;
+                    let newFilter: IFilter[];
+                    if (checked) {
+                        newFilter = [...localselectedFilters, filter];
+                    } else {
+                        newFilter = localselectedFilters.filter((f) => f.id != filter.id);
+                        setLocalSelectedFilters(newFilter);
+                    }
+
+                    handleOnChange(newFilter);
+                }}
+                type="checkbox"
+            >
+                <Text>{getMappedName(filter)}</Text>
+            </Checkbox>
+            <Tag className={styles.tag}>{numberFormat(filter.data.count)}</Tag>
+        </StackLayout>
+    );
+
+    const noDataFilter = noDataInputOption && filteredFilters.find((f) => f.id === ArrangerValues.missing);
```

Comment:
I would use `some` vs `find` if you are looking only for a boolean. Find returns the found object if it exists.

Reply:
I want the object :)